sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_single_node/language-modeling.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    hyperparameters["per_device_train_batch_size"] = 32
</pre><p>This is less than than the native batch size (which is 48), and it does not seem right. With the compiler, you should be able to use a larger batch size.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Addressed in another comment

Reply:
Pardon, which comment?